From 69789f2a780a0a1a2d16e42ec3788c7b844fe1e9 Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Tue, 4 Oct 2016 19:46:58 +0200
Subject: [PATCH] Make the vdso32 also build big-endian
Patch-mainline: Never, not needed
References: bsc#967716

The vdso32 is built LE on LE system and fails to link.

Upstream probably takes care of this in the endian-switch infrastructure which
we don't have.

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
---
 arch/powerpc/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/powerpc/Makefile b/arch/powerpc/Makefile
index 1934707..9bd0223 100644
--- a/arch/powerpc/Makefile
+++ b/arch/powerpc/Makefile
@@ -22,7 +22,7 @@ CROSS32AR		:= $(CROSS32_COMPILE)ar
 
 ifeq ($(HAS_BIARCH),y)
 ifeq ($(CROSS32_COMPILE),)
-CROSS32CC	:= $(CC) -m32
+CROSS32CC	:= $(CC) -m32 -mbig-endian
 CROSS32AR	:= GNUTARGET=elf32-powerpc $(AR)
 endif
 endif
-- 
1.8.5.6

